mlflow/genai/scorers/builtin_scorers.py

```diff
@@ -41,6 +53,15 @@ def __call__(self, **kwargs):
         else:
             raise ValueError("This scorer isn't recognized since it doesn't have a name.")
 
+    @abstractmethod
+    def with_config(self, **kwargs) -> "_BaseBuiltInScorer":
+        """
+        Get a new scorer instance with the given configuration, such as name, global guidelines.
```

Comment:
It might be worthwhile to add an example of overriding a built-in scorer in the API docs. Can one be added here?